apps/server/src/shared/common/loggable-exception/feature-disabled.loggable-exception.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { FeatureDisabledLoggableException } from './feature-disabled.loggable-exception';
+
+describe(FeatureDisabledLoggableException.name, () => {
```

Comment:
Good idea btw.. Maybe this loggable is something for the software development channel?